### PR TITLE
Update odo commands output and remove outdated info.

### PR DESCRIPTION
**Pull request by yhontyk**
Updates the output of the commands.
Branch 4.5 and 4.6

https://github.com/openshift/odo/issues/3878

### CODE REVIEW

File: modules/developer-cli-odo-creating-services-from-yaml-files.adoc [hunk 1]
```diff
@@ -14,14 +14,14 @@ Printing and modifying the YAML used to start a service
 +
 [source,terminal]
 ----
-$ odo service create <operator-name> --crd <cr-name> --dry-run
+$ odo service create <operator-name>  --dry-run
```
Comment (lamek) [inline modules/developer-cli-odo-creating-services-from-yaml-files.adoc:17]: NIT - extra space between` <operator-name>` and --dry-run.

**Review by lamek — COMMENTED** [review]
One NIT and otherwise LGTM.

File: modules/developer-cli-odo-creating-services-from-yaml-files.adoc (hunk 1 above)
Comment (boczkowska) [reply modules/developer-cli-odo-creating-services-from-yaml-files.adoc]: @lamek done! Thank you.

**Pull request merged**